### ISSUE TITLE
Detected

### ISSUE THREAD

**Issue by localhosti** [icyguider/Shhhloader#6]
All injection method detected Defender

**Comment by icyguider** [comment]
Hi there. This is something I've noticed as well recently. It looks to be related to Defender's "Cloud-delivered protection" feature, as most methods don't seem to get detected while that is turned off.

At the moment I am unaware of how to mitigate this. I might try to research ways to specifically bypass Defender's cloud-delivered protection, or try to implement different injection methods to see if those would make a difference.

I did recently test this tool against CrowdStrike's EDR using the ProcessHollow and QueueUserAPC injection techniques and it was not blocked, so the current injection methods certainly still hold some value, just perhaps not against Defender. If you have any suggestions on how these detections might be circumvented, I am definitely interested to hear and learn! :)
